modules/rgbd/src/tsdf.cpp

```diff
@@ -473,7 +536,8 @@ inline TsdfType TSDFVolumeCPU::interpolateVoxel(const v_float32x4& p) const
     for(int i = 0; i < 8; i++)
         vx[i] = volData[neighbourCoords[i] + coordBase].tsdf;
 
-    v_float32x4 v0246(vx[0], vx[2], vx[4], vx[6]), v1357(vx[1], vx[3], vx[5], vx[7]);
+    v_float32x4 v0246 = tsdfToFloat_INTR(v_int32x4(vx[0], vx[2], vx[4], vx[6]));
+    v_float32x4 v1357 = tsdfToFloat_INTR(v_int32x4(vx[1], vx[3], vx[5], vx[7]));
```

Comment:
Please replace these places by `v_load_deinterleave` as in `hash_tsdf.cpp` file

Reply:
We  have float at start in `hash_tsdf.cpp` , but in `tsdf.cpp` we have int_8t